src/trex/clone.py

```diff
@@ -169,9 +189,16 @@ def dot(self, highlight=None):
         for node in self._graph.nodes():
             if self._graph.neighbors(node):
                 width = int(1 + node_scaling * math.log(node.n))
-                intersection = set(node.cell_ids) & highlight
-                hl = ",fillcolor=yellow" if intersection else ""
-                hl_label = f" ({len(intersection)})" if intersection else ""
+                intersection = set(node.cell_ids) & highlight_cell_ids
+                if node in highlight_nodes:
```

Comment:
I’ll change this. The initial idea was to use a generic name because you could in principle use it to highlight any set of nodes. But then I later added the "(doublet)" text, which makes the highlighting specific to doublets, so it’s probably better to name the parameter `doublets`.